js/foam/core/types/EnumPropertyTrait.js

```diff
@@ -29,6 +29,12 @@ CLASS({
     {
       name: 'view',
       defaultValue: 'foam.ui.ChoiceView'
-    }
+    },
+    {
+      name: 'toPropertyE',
+      defaultValue: function(X) {
+        return X.foam.u2.Select.create({ prop: this }, X);
```

Comment:
Doing X.foam.u2.Selected instead of X.lookup() will prevent the overriding of Select with registerModel.  Just add a comment and we'll fix when we have some kind of FutureElement.